ksql-engine/src/main/java/io/confluent/ksql/structured/SchemaKStream.java

```diff
@@ -614,5 +653,6 @@ private void fillWithNulls(final List<Object> columns, final int numToFill) {
       }
     }
   }
-
 }
+// CHECKSTYLE_RULES.ON: ParameterNumber
+// CHECKSTYLE_RULES.ON: ClassDataAbstractionCoupling
```

Comment:
Disable rules around the smallest amount of code possible. i.e. ParameterName only around the constructor(s) that need it and `ClassDataAbstractionCoupling` just around the class name.

This stops / deters other methods or nested classes being added with equally bad practice ;)